runtime/frequency/src/lib.rs

```diff
@@ -373,6 +373,43 @@ impl pallet_balances::Config for Runtime {
 	type ReserveIdentifier = [u8; 8];
 }
 
+parameter_types! {
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(10) * RuntimeBlockWeights::get().max_block;
```

Comment:
Would you please add some comments about what is this weight for and how it's been used?